packages/mobile/src/transactions/__snapshots__/TransferConfirmationCard.test.tsx.snap

```diff
@@ -108,17 +108,26 @@ exports[`TransferConfirmationCard renders correctly for CELO withdrawal transact
                   </svg>
                 </View>
               </View>
-              <Text
+              <View
```

Comment:
This changes are due to the change to the implementation of `Expandable`. It changes nothing visually, it's just wrapping the `Text` with a `View`.